WPMedia/fusion-news-theme-blocks
# TMEDIA-197 - Extract common components from promos

## Description

Made reusable components and implemented reusable component into promo blocks

## Jira Ticket
- [TMEDIA-197](https://arcpublishing.atlassian.net/browse/TMEDIA-197)

## Acceptance Criteria
* Extract reused code from promos into shared styles and update promo blocks to use shared components

## Test Steps

1. Checkout this branch `git checkout TMEIDA-197-extract-common-components-from-promos`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/extra-large-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-promo-block,@wpmedia/small-promo-block,@wpmedia/extra-large-manual-promo-block,@wpmedia/large-manual-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/small-manual-promo-block,@wpmedia/shared-styles`
3.  Verify promo blocks didn't change - test page - http://localhost/pf/test-all-promos/?_website=the-gazette


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
